crate/multiworld-gui/src/main.rs

```diff
@@ -1689,9 +1710,17 @@ impl State {
                             if rooms.is_empty() {
                                 Text::new("(no rooms currently open)").into()
                             } else {
-                                let mut rooms = rooms.iter().map(|(&id, (name, password_required))| RoomFormatter { id, name: name.clone(), password_required: password_required.clone() }).collect_vec();
+                                let mut rooms = rooms.iter().map(|(&id, (name, password_required))| RoomFormatter { id, name: name.clone(), password_required: password_required.clone(), active: true }).filter(|room| room.name.to_lowercase().contains(&self.room_filter.to_lowercase())).collect_vec();
                                 rooms.sort();
-                                PickList::new(rooms, existing_room_selection.clone(), Message::SetExistingRoomSelection).into()
+                                if rooms.is_empty() {
+                                    rooms.push(RoomFormatter { password_required: false, name: String::from("No room found"), id: 0, active: false });
+                                }
+                                let mut stack = Stack::new().width(360.0);
```

Comment:
Does this work correctly when resizing the window?

Reply:
short answer is no it does not.
But right now the picklist does not work with resizing anyway, so it gave the same behaviour
I added the width since the picklist was scaling with the width of the largest option in the list.
see those image for reference (top is this branch and bottom is latest released version)
![image](https://github.com/user-attachments/assets/421ae0e7-f420-4328-9d87-606cae8df0b8)
![image](https://github.com/user-attachments/assets/77d046c4-6694-4c38-b6c0-71726dfdb817)

Reply:
Oh I never actually noticed that. I guess that's fine for now then.